Jenkinsfile

```diff
@@ -0,0 +1,203 @@
+                        [$class: 'CleanBeforeCheckout'],
+                    ],
+
+                )
+            }
+        }
+        stage('clean') {
+            when {
+                expression { return params.CLEAN }
+            }
+            steps {
+                sh '''
+                    docker images --format="{{.Repository}}:{{.Tag}}" | \
+                        grep -E '/scf-|/uaa-|^uaa-role-packages:|^scf-role-packages:' | \
+                        xargs --no-run-if-empty docker rmi
+                '''
+            }
+        }
+        stage('checkout') {
+            when {
+                expression { return ! params.SKIP_CHECKOUT }
+            }
+            steps {
+                sh '''
+                    git config --global --replace-all submodule.fetchJobs 0
+                '''
+                checkout scm
+            }
+        }
+        stage('tools') {
+            steps {
+                ansiColor('xterm') {
+                    sh '''
+                        set -e +x
+                        source ${PWD}/.envrc
+                        set -x
+                        unset HCF_PACKAGE_COMPILATION_CACHE
+                        make ${FISSILE_BINARY}
+                    '''
+                }
+            }
+        }
+        stage('build') {
+            steps {
+                ansiColor('xterm') {
+                    sh '''
+                        set -e +x
+                        source ${PWD}/.envrc
+                        set -x
+                        unset HCF_PACKAGE_COMPILATION_CACHE
+                        make vagrant-prep validate
+                    '''
+                }
+            }
+        }
+        stage('dist') {
+            steps {
+                ansiColor('xterm') {
+                    sh '''
+                        set -e +x
+                        source ${PWD}/.envrc
+                        set -x
+                        unset HCF_PACKAGE_COMPILATION_CACHE
+                        rm scf-*-amd64-*.zip
+                        make helm bundle-dist
+                    '''
+                }
+            }
+        }
+        stage('publish') {
+            when {
+                expression { return params.PUBLISH }
+            }
+            steps {
+                ansiColor('xterm') {
+                    withCredentials([usernamePassword(
+                        credentialsId: params.DOCKER_CREDENTIALS,
+                        usernameVariable: 'DOCKER_HUB_USERNAME',
+                        passwordVariable: 'DOCKER_HUB_PASSWORD',
+                    )]) {
+                        sh 'docker login -u "${DOCKER_HUB_USERNAME}" -p "${DOCKER_HUB_PASSWORD}" '
+                    }
+                    sh '''
+                        set -e +x
+                        source ${PWD}/.envrc
+                        set -x
+                        unset HCF_PACKAGE_COMPILATION_CACHE
```

Comment:
I just _really_ don't want to copy the compile cache anywhere (I'm not in a VM!)